rlmodule/skrl/torch/gymnasium/pendulum_ppo_rnn_shared_model.py

```diff
@@ -14,9 +14,9 @@
 import torch.nn as nn
 
 from rlmodule.skrl.torch import SharedRLModelCfg, build_model
-from rlmodule.skrl.torch.network import MlpCfg
+from rlmodule.skrl.torch.log_std import ParameterLogStdCfg
```

Comment:
Please just make it import all from output layer
from rlmodule.skrl.torch.output_layer import DeterministicLayerCfg, GaussianLayerCfg, ParameterLogStdCfg

Reply:
Did you forget to push your last commit? 

because here you say "done" but it was not done:

![image](https://github.com/user-attachments/assets/53f7c3d8-87d1-4afe-a1a4-0c2cb09aa03f)

Reply:
Ahh, my takeway was that we should fix the bad imports and always do rlmodule.skrl :D  now I see